src/main/java/me/botsko/prism/Prism.java

```diff
@@ -332,7 +332,7 @@ public void onEnable() {
             monitor.addGauge(Prism.class, recordingQ, "RecordingQueueSize");
             ActionMeter.setupActionMeter(monitor.getRegistry());
         }
-        log("Initializing Prism " + plugin_version + ". By Viveleroi.");
+        log("Initializing Prism " + plugin_version + ". maintained my the AddstarMC Network");
```

Comment:
I think you meant `by`, not `my`.

Reply:
:+1: Good catch, thanks!

Reply:
Fixed via https://github.com/AddstarMC/Prism-Bukkit/pull/146